### PR TITLE
Migration/keysignote

### PR DESCRIPTION
**Pull request by rvilarl**
keysignote is there!

### CODE REVIEW

File: src/keysignote.ts [hunk 1]
```diff
@@ -15,28 +18,32 @@ export class KeySigNote extends Note {
     this.ignore_ticks = true;
   }
 
-  getBoundingBox() {
+  getBoundingBox(): BoundingBox | undefined {
```
Comment (0xfe) [inline src/keysignote.ts:21]: I think we can remove this whole method, now?. It just calls the parent.

**Review by 0xfe — COMMENTED** [review]
Looks good. Minor comment.

File: src/keysignote.ts (hunk 1 above)
Comment (rvilarl) [reply src/keysignote.ts]: done

**Comment by 0xfe** [comment]
👍

**Pull request merged**